cmd.go

```diff
@@ -526,3 +526,15 @@ func passToMakepkg(dir string, args ...string) (err error) {
 	}
 	return
 }
+
+func passToGit(dir string, _args ...string) (err error) {
```

Comment:
Maybe pick a different name for this `_args` variable? I skimmed quickly through the code and didn't find other variables named similarly (starting with an underline).

Reply:
I would rather not have to have two separate variables at all like in the other passTo* functions but it didn't work like that. If you've got a better method or name feel free to suggest one.